pkgs/misc/uq/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv
```

Comment:
You can drop `stdenv` add it isn't needed with `lib`.

Reply:
:+1:.